tests/Testcontainers.Tests/Unit/Containers/Unix/Modules/Databases/MySqlTestcontainerNormalUserTest.cs

```diff
@@ -0,0 +1,70 @@
+namespace DotNet.Testcontainers.Tests.Unit
```

Comment:
Can we put the tests into a shared static class?

```csharp
public static class MySqlTestcontainerTest
{
  public sealed class MySqlRootUsername : IClassFixture<...>
  {
  }

  public sealed class MySqlCustomUsername : IClassFixture<...>
  {
  }
}
````

Reply:
yes 
8bd1b53